modules/mobile/app/helpers/mobile/facilities_helper.rb

```diff
@@ -105,6 +107,7 @@ def phone_from_facility(facility)
     # Returns the distance between these two
     # points in either miles or kilometers
     def haversine_distance(geo_a, geo_b, miles: true)
+      # not yet in use. could be helpful when we turn this on
       Rails.logger.info('haversine_distance coords', geo_a, geo_b)
```

Comment:
Category: data discovery
Action: leave for now because this code is not yet being used and may be helpful if/when this is rolled out